pkg/hsskey/hsskey.go

```diff
@@ -182,6 +194,38 @@ func GetAllHss(warnings io.Writer, verboseDangerous bool, eepromPattern string,
 	return nil, fmt.Errorf("failed all HSS retrieval attempts")
 }
 
+// evaluateHssPath evaluates a filepath to return the file or contents if a directory.
+// This function evaluates the base pointer for symlinks.
+func evaluateHssPath(hssFiles string) ([]string, error) {
+	// Follow symlink if needed.
+	hssFiles, err := filepath.EvalSymlinks(hssFiles)
+	if err != nil {
+		return nil, fmt.Errorf("could not eval symlink %s", hssFiles)
```

Comment:
This is not actually right. There may be no symlinks in the path. I'd suggest 
fmt.Errorf("%q:%w", hssFiles, err)

Reply:
The "%q:%w" ended up causing problems on some of the older automated testing. 

_fmt.Errorf("failed evaluating HSS path: %w", err)_ should be equally fine here.